src/Spaceport/Commands/InitCommand.php

```diff
@@ -33,109 +36,31 @@ protected function configure()
      */
     protected function doExecute(InputInterface $input, OutputInterface $output)
     {
+        $symfonyVersion = $input->getArgument("symfony_version");
+        if ($symfonyVersion == "sf3") {
```

Comment:
Now I'm looking at it, maybe we should do this the same as the other questions like so:
`$this->io->choice()`